_tsinfermodule.c

```diff
@@ -547,7 +547,7 @@ TreeSequenceBuilder_add_path(TreeSequenceBuilder *self, PyObject *args, PyObject
         goto out;
     }
     shape = PyArray_DIMS(right_array);
-    if (shape[0] != (npy_intp) num_edges) {
```

Comment:
Looks like something happened during rebase here - I think I added these casts a few days ago to get rid of compiler warnings.

Reply:
Yeah I spotted it too - think it was while I was messing around trying to find the segfault. Should be all cleared up now as I hard reset and re-ran pre-commit.